playwright.config.js

```diff
@@ -5,7 +5,8 @@ export default defineConfig({
         {
             name: 'windows',
             testMatch: [
-                'integration-test/playwright/duckplayer.spec.js'
+                'integration-test/playwright/duckplayer.spec.js',
+                'integration-test/playwright/harmful-apis.spec.js'
```

Comment:
Ahh! this made me realise - we accidentally excluded the original tests for windows permissions in this commit: 02e53bb9dc9c92e42a86a6aebaa0cc32ada1c002 oops! I'll create a separate PR for that - but it makes me think if there's some redundant overlap? 

Perhaps on your branch you could re-enable the windows-permissions tests and evaluate if your work makes any of it obsolete?

Reply:
Yeah, I did actually move some code from the windows permissions in this PR. Re-enabled the test again, it still works well